fake_joint_driver/include/fake_joint_driver/fake_joint_driver.h

```diff
@@ -7,16 +7,18 @@
  * FakeJointDriver class (only do loopback from command to status)
  * derived from the hardware_interface class
  */
-#include <hardware_interface/joint_command_interface.h>
-#include <hardware_interface/joint_state_interface.h>
-#include <hardware_interface/robot_hw.h>
+#include <rclcpp/rclcpp.hpp>
 
-class FakeJointDriver : public hardware_interface::RobotHW
+#include <hardware_interface/joint_command_handle.hpp>
+#include <hardware_interface/joint_state_handle.hpp>
+#include <hardware_interface/robot_hardware.hpp>
+
+class FakeJointDriver : public hardware_interface::RobotHardware
 {
 private:
-  hardware_interface::JointStateInterface joint_state_interface;
-  hardware_interface::PositionJointInterface position_joint_interface;
-  hardware_interface::VelocityJointInterface velocity_joint_interface;
+  hardware_interface::JointStateHandle joint_state_interface;
+  // hardware_interface::PositionJointInterface position_joint_interface;
```

Comment:
Aren't these interfaces available yet?

Reply:
Unfortunately, they are not available yet

Reply:
I thought the joint_command_handle as the direct replacement for the joint_command_interface is everything we need here. According to documentation that's enough for running the trajectory controller. I would remove the commented interfaces and just add the JointCommandHandle.